### PR TITLE
Add WOFF2 support

### PR DESCRIPTION
**Pull request by paulschreiber**
Fixes https://github.com/kenwheeler/slick/issues/995

### CODE REVIEW

File: slick/fonts/slick.woff2
[the remark this answers is not in the document]
Comment (markvantilburg) [reply slick/fonts/slick.woff2:1]: i validated the woff2 file using:

`<style>
@font-face
{
    font-family: 'slick';
    font-weight: normal;
    font-style: normal;
    src:url('slick.woff') format('woff');
}

@font-face
{
    font-family: 'slick2';
    font-weight: normal;
    font-style: normal;
    src:url('slick.woff2') format('woff2');
}
</style>
<div style="font-family: 'slick2';">
WOFF2
→
←
•
</div>

<div style="font-family: 'slick';">
WOFF
→
←
•
</div>`

[the remark this answers is not in the document]
Comment (markvantilburg) [reply slick/fonts/slick.woff2:1]: Screenshot 2024-07-10 144120

**Comment by markvantilburg** [comment]
@ahmadalfy This looks good to me

**Review by markvantilburg — APPROVED** [review]

**Review by ahmadalfy — APPROVED** [review]
Thanks @markvantilburg for the confirmation. This one is ready

**Pull request merged**